lib/Epub/RubbishHtmlParser/RubbishHtmlParser.cpp

```diff
@@ -244,3 +246,27 @@ void RubbishHtmlParser::render_page(int page_index, Renderer *renderer, Epub *ep
   // renderer->fill_rect(0, 0, renderer->get_page_width(), renderer->get_page_height(), 0xFF);
   pages[page_index]->render(renderer, epub);
 }
+
+string RubbishHtmlParser::entityParser(string text)
+{
+  string res = "";
+  for (int i = 0; i < text.size(); ++ i) {
+      bool flag = false;
+      for (auto it = begin(convert); it != end(convert); ++ it) {
```

Comment:
We can probably optimise this by only doing the check if we see an "&" character - all HTML entities will start with &.

Do we need the unordered map? We could just use the array of HTML entities - for the ones where the entire is en empty string we should probably just remove them as they should never really occur.